components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/pom.xml

```diff
@@ -116,6 +120,18 @@
             <artifactId>slf4j-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>2.8.9</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.felix</groupId>
+            <artifactId>org.osgi.foundation</artifactId>
+            <version>1.0.0</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Move dependency version tags to root pom. 
In root pom dependency version should be in ${abc.version} format.

Reply:
Fixed with 30d0ecb9c29c0024fa182ba597c8eda4c128c451